src/client.rs

```diff
@@ -11,29 +11,29 @@ use thiserror::Error;
 
 use std::path::PathBuf;
 
+use super::enums::{
+    AdditionalType, AlbumType, Country, IncludeExternal, RepeatState, SearchType, TimeRange, Type,
+};
 use super::http::{BaseClient, Query};
 use super::json_insert;
-use super::model::album::{FullAlbum, FullAlbums, PageSimpliedAlbums, SavedAlbum, SimplifiedAlbum};
-use super::model::artist::{CursorPageFullArtists, FullArtist, FullArtists};
-use super::model::audio::{AudioAnalysis, AudioFeatures, AudioFeaturesPayload};
-use super::model::category::PageCategory;
-use super::model::context::{CurrentlyPlaybackContext, CurrentlyPlayingContext};
-use super::model::cud_result::CUDResult;
-use super::model::device::DevicePayload;
-use super::model::page::{CursorBasedPage, Page};
-use super::model::playing::{PlayHistory, Playing};
-use super::model::playlist::{FeaturedPlaylists, FullPlaylist, PlaylistTrack, SimplifiedPlaylist};
-use super::model::recommend::Recommendations;
-use super::model::search::SearchResult;
-use super::model::show::{
+use super::model::CUDResult;
```

Comment:
I think this is one of the few acceptable moments where you can use `use super::model::*` and `use super::enums::*`. The client should be using all of them anyway, and it cleans up the header a lot. What do you think? That, or merging all the `use super::model` into a huge single import. I prefer the first option, though.

Reply:
Yep, it's a acceptable choice to use `use super::model::*`, or we have to import a lot of model files individually.